packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesPackage.kt

```diff
@@ -119,5 +120,7 @@ class UpdatesPackage : Package {
 
   companion object {
     private val TAG = UpdatesPackage::class.java.simpleName
+    val useNativeDebug = BuildConfig.EX_UPDATES_NATIVE_DEBUG
+    val useCustomInit = BuildConfig.EX_UPDATES_CUSTOM_INIT
```

Comment:
maybe use `internal` or `private` here to prevent exposing these flags

Reply:
For custom updates flow, the `useNativeDebug` flag does need to be public. However, we can make `useCustomInit` an internal flag.